cmd/cluster-agent/commands/clusterchecks.go

```diff
@@ -45,13 +48,14 @@ func GetClusterChecksCobraCmd(flagNoColor *bool, confPath *string, loggerName co
 				return err
 			}
 
-			if err = flare.GetClusterChecks(color.Output); err != nil {
+			if err = flare.GetClusterChecks(color.Output, checkName); err != nil {
 				return err
 			}
 
 			return flare.GetEndpointsChecks(color.Output)
 		},
 	}
+	clusterChecksCmd.PersistentFlags().StringVarP(&checkName, "cluster-check-name", "", "", "the cluster check name to filter for")
```

Comment:
nit: I think `agent clusterchecks --cluster-check-name foo` is a bit redundant. 
`agent clusterchecks --check-name foo` or `agent clusterchecks --check foo` can be better

Reply:
The idea was that this is only for cluster check and not for endpoint checks. But I am not 100% deep into that topic, so I am open for renaming it 👍

Reply:
I don't see why this cannot support endpoint checks, happy to work on supporting them later.

Reply:
I am not sure about the semantics and UX. Are we fine if we filter for 
`orchestrator, that this means we filter for `endpoints` and for `clusterchecks`? If yes, I can try to make it happen here (adding that to the method `GetEndpointsChecks`). But in case that would mean larger changes, then I suggest doing it in another PR 🤔

Reply:
Currently `agent clusterchecks` lists both cluster and endpoint checks, so if we'll add a filtering option it should apply to both cluster and endpoint checks imho.

Reply:
Added in 6fa7daee67b9cbca67ed2ebf0ab8e8229eb54cd1 👍